src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -281,6 +282,7 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         const { chart } = this;
         chart.on("click", DEFAULT_LAYER_NAME, this.handleMapClick);
         chart.on("load", this.setupMap);
+        chart.on("load", this.adjustChartHeight);
```

Comment:
Could we put `adjustChartHeight` into `setupMap`?

Reply:
I don't think so, `setupMap` is now having bunch of logic.
Separation is better, IMO.